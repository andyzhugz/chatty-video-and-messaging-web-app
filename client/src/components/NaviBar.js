import '../stylesheets/NaviBar.css';
const NaviBar = () => {
  return (
    <div>
      <ul id="nav_ul">
        <li className="nav_li">
          <a href="/profile">Profile</a>
        </li>
        <li className="nav_li">
          <a href="/main">Messaging</a>
        </li>
        <li className="nav_li">
          <a href="/status">Status</a>
        </li>
      </ul>
    </div>
  );
};

export default NaviBar;
