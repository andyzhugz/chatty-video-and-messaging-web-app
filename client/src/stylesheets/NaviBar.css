#nav_ul {
  list-style-type: none;
  margin: 0;
  padding: 0;
  overflow: hidden;
  background-color: #333;
}

.nav_li {
  float: left;
  width: 200px;
}

.nav_li a {
  display: block;
  color: white;
  text-align: center;
  padding: 14px 16px;
  text-decoration: none;
}

.nav_li a:hover {
  background-color: #111;
}
