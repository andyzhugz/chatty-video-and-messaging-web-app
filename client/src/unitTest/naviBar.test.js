import React from 'react';
import NaviBar from '../components/NaviBar.js';

const renderer = require('react-test-renderer');

// snapshot testings
describe('Test snapshot', () => {
  test('Test Profile', () => {
    const component = renderer.create(<NaviBar />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
