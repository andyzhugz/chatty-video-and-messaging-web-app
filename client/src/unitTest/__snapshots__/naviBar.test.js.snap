// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot Test Profile 1`] = `
<div>
  <ul
    id="nav_ul"
  >
    <li
      className="nav_li"
    >
      <a
        href="/profile"
      >
        Profile
      </a>
    </li>
    <li
      className="nav_li"
    >
      <a
        href="/main"
      >
        Messaging
      </a>
    </li>
    <li
      className="nav_li"
    >
      <a
        href="/status"
      >
        Status
      </a>
    </li>
    <li
      className="nav_li"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Logout
      </a>
    </li>
    <li
      className="nav_li"
    >
      <a
        href="/main"
      >
        
      </a>
    </li>
  </ul>
</div>
`;
